### PR TITLE
Add workload ocp4-workload-quarkus-super-heroes-demo create new Quarkus Superheroes Demo catalog item

### PR DESCRIPTION
**Pull request by edeandrea**
##### SUMMARY

This is a new Demo catalog item for the Quarkus Superheroes Demo. This coincides with https://issues.redhat.com/browse/GPTEINFRA-3573

##### ISSUE TYPE

- New role Pull Request

##### COMPONENT NAME
quarkus-super-heroes-demo

##### ADDITIONAL INFORMATION
See https://issues.redhat.com/browse/GPTEINFRA-3573 for complete information about the new catalog item.

This role does not require any additional variables to be passed to it.

I tested this by ordering an OpenShift 4.10 workshop and doing the following:

```shell
WORKLOAD="ocp4-workload-quarkus-super-heroes-demo"
OCP_USERNAME="opentlc-mgr"
GUID=<MY_ENV_GUID>
SANDBOX_NUMBER=<MY_SANDBOX_NUMBER>
TARGET_HOST="bastion.${GUID}.sandbox${SANDBOX_NUMBER}.opentlc.com"
SSH_KEY=~/.ssh/id_rsa
ANSIBLE_USER=lab-user

ansible-playbook -i ${TARGET_HOST}, ./configs/ocp-workloads/ocp-workload.yml \
                    -e"ansible_ssh_private_key_file=${SSH_KEY}" \
                    -e"ansible_user=${ANSIBLE_USER}" \
                    -e"ocp_workload=${WORKLOAD}" \
                    -e"silent=False" \
                    -e"ACTION=create" \
                    -e"ocp_username=${OCP_USERNAME}" \
                    -e"guid=${GUID}" \
                    -e"subdomain_base=${TARGET_HOST}"
```

### CODE REVIEW

**Comment by edeandrea** [comment]
I'm finding that the yamllint is failing on lines too long in some of these. Many of these (grafana dashboards especially) are embedding yaml & json documents as text in other yaml documents and its either impossible to insert line breaks or doing so would make it far less readable.

There are other instances as well, like

```yaml
loop: "{{ ocp4_workload_quarkus_superheroes_java_versions | product(ocp4_workload_quarkus_superheroes_kinds) | list }}"
```

Where I would argue that re-writing as

```yaml
loop: >-
  "{{ ocp4_workload_quarkus_superheroes_java_versions | product(ocp4_workload_quarkus_superheroes_kinds) | list }}"
```

just to satisfy a linter makes it less readable.

Similarly when using the [`uri` module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/uri_module.html), the documentation of the `validate_certs` option says that values should be `yes`/`no`, but the linter complains that it should be `true`/`false`.

File: ansible/roles/ocp4-workload-quarkus-super-heroes-demo/.yamllint [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (wkulhanek) [inline ansible/roles/ocp4-workload-quarkus-super-heroes-demo/.yamllint:1]: This should be removed. If a line will break yamllint it needs to be fixed - single line exceptions are acceptable. But not a blanket change.

File: ansible/roles/ocp4-workload-quarkus-super-heroes-demo/tasks/post_workload.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      - {{ t_project_name }} ({{ console_url }}/topology/ns/{{ t_project_name }})"
+        - Super Heroes UI: http://ui-super-heroes-{{ t_project_name }}.{{ route_subdomain }}
+        - Event statistics UI: http://event-statistics-{{ t_project_name }}.{{ route_subdomain }}
+        - Heroes service data UI: http://rest-heroes-{{ t_project_name }}.{{ route_subdomain }}
+        - Villains service data UI: http://rest-villains-{{ t_project_name }}.{{ route_subdomain }}
+        - Apicurio Schema Registry: http://apicurio-{{ t_project_name }}.{{ route_subdomain }}
+        - Prometheus: http://prometheus-operated-{{ t_project_name }}.{{ route_subdomain }}
+
+  loop: "{{ java_versions | product(app_kinds) | list }}"
+  when: not silent|bool
```
Comment (wkulhanek) [inline ansible/roles/ocp4-workload-quarkus-super-heroes-demo/tasks/post_workload.yml:35]: Same here.

File: ansible/roles/ocp4-workload-quarkus-super-heroes-demo/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+become_override: false
+ocp_username: opentlc-mgr
+ocp_password: 'r3dh4t1!'
+silent: false
+install_prometheus: true
```
[the remark this answers is not in the document]
Comment (edeandrea) [reply ansible/roles/ocp4-workload-quarkus-super-heroes-demo/defaults/main.yml]: This has been fixed.

File: ansible/roles/ocp4-workload-quarkus-super-heroes-demo/.yamllint (hunk 1 above)
Comment (edeandrea) [reply ansible/roles/ocp4-workload-quarkus-super-heroes-demo/.yamllint]: This has been removed.

File: ansible/roles/ocp4-workload-quarkus-super-heroes-demo/tasks/post_workload.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+- name: output demo info
+  agnosticd_user_info:
+    msg: |
+      OpenShift credentials: {{ ocp_username }} / {{ ocp_password }}
+      OpenShift/Kubernetes API (use with oc login): {{ master_url }}
+      OpenShift Console URL: {{ console_url }}
+
+      There are 4 projects, each correlating to a version of the application as defined in {{ docs_link }}#versions:
+
+  when: not silent|bool
```
[the remark this answers is not in the document]
Comment (edeandrea) [reply ansible/roles/ocp4-workload-quarkus-super-heroes-demo/tasks/post_workload.yml]: This has been fixed.

**Comment by edeandrea** [comment]
@wkulhanek I've made the requested changes. I also moved the workload from the `roles` directory to the `roles_ocp_workloads` directory.

**Pull request merged**